abnormal_security/plugin.spec.yaml

```diff
@@ -0,0 +1,227 @@
+      description: Return path
+      type: string
+      required: false
+    replyToEmails:
+      title: Reply to Emails
+      description: Reply to emails
+      type: '[]string'
+      required: false
+    ccEmails:
+      title: CC Emails
+      description: CC emails
+      type: '[]string'
+      required: false
+    senderIpAddress:
+      title: Sender IP Address
+      description: Sender IP address
+      type: string
+      required: false
+    impersonatedParty:
+      title: Impersonated Party
+      description: Impersonated party
+      type: string
+      required: false
+    attackVector:
+      title: Attack Vector
+      description: Attack vector
+      type: string
+      required: false
+    attachmentNames:
+      title: Attachment Names
+      description: Attachment names
+      type: '[]string'
+      required: false
+    summaryInsights:
+      title: Summary Insights
+      description: Summary insights
+      type: '[]string'
+      required: false
+    remediationTimestamp:
+      title: Remediation Timestamp
+      description: Remediation timestamp
+      type: string
+      required: false
+    isRead:
+      title: Is Read
+      description: Is Read
+      type: boolean
+      required: false
+    attackedParty:
+      title: Attacked Party
+      description: Attacked party
+      type: string
+      required: false
+    abxMessageId:
+      title: ABX Message ID
+      description: ABX Message ID
+      type: integer
+      required: false
+    abxPortalUrl:
+      title: ABX Portal URL ID
+      description: ABX portal URL ID
+      type: string
+      required: false
+    subject:
+      title: Subject
+      description: Subject
+      type: string
+      required: false
+    fromAddress:
+      title: From Address
+      description: From address
+      type: string
+      required: false
+    fromName:
+      title: From Name
+      description: From name
+      type: string
+      required: false
+    toAddresses:
+      title: To Address
+      description: To address
+      type: string
+      required: false
+    recipientAddress:
+      title: Recipient Address
+      description: Recipient address
+      type: string
+      required: false
+    receivedTime:
+      title: Received Time
+      description: Received time
+      type: string
+      required: false
+    sentTime:
+      title: Sent Time
+      description: Sent time
+      type: string
+      required: false
+    internetMessageId:
+      title: Internet Message ID
+      description: Internet message ID
+      type: string
+      required: false
+    urls:
+      title: URLs
+      description: URLs
+      type: '[]string'
+      required: false
+  threat_details:
+    threatId:
+      title: Threat ID
+      description: Threat ID
+      type: string
+      required: false
+    messages:
+      title: Messages
+      description: List of messages
+      type: '[]message'
+      required: false
+connection:
+  hostname:
```

Comment:
We should change this to `url` since it's a URL and not a hostname, and then update all the references.